trade-finance-manager-ui/server/api-response-types/fee-record-correction-response-body.ts

```diff
@@ -0,0 +1,3 @@
+export type FeeRecordCorrectionResponseBody = {
```

Comment:
Would you mind renaming this to either `PostFeeRecordCorrectionResponseBody` or `CreateFeeRecordCorrectionResponseBody`? Have a see what lines up best with existing names and the endpoint in question.
We are going to be adding a get endpoint at some point in the near future for clicking into a correction on the history tab and I think this could get confusing at that point so best to be specific now.